src/arviz_stats/base/diagnostics.py

```diff
@@ -760,3 +760,138 @@ def _r2_score(y_true, y_pred):
         var_e = np.var(y_true - y_pred, axis=1)
         r_squared = var_y_est / (var_y_est + var_e)
         return r_squared
+
+    @staticmethod
+    def _mae(observed, predicted):
+        """Compute the Mean Absolute Error (MAE).
+
+        Parameters
+        ----------
+        observed: array-like of shape = (n_outputs,)
+            Ground truth (correct) target values.
+        predicted: array-like of shape = (n_outputs)
+            Estimated target values.
+
+        Returns
+        -------
+        tuple (mean, std_error)
+            mean: float
+                Mean absolute error.
+            std_error: float
+                Standard error of the mean absolute error.
```

Comment:
```suggestion
        mean: float
            Mean absolute error.
        std_error: float
            Standard error of the mean absolute error.
```

these can be directly documented like this